.gitignore

```diff
@@ -50,8 +48,3 @@ packages/**/*.map
 # cache
 .eslintcache
```

Comment:
Do not touch this file, if you need cleanup or found unnecessary things, let's do it in the separate PR